src/components/editor/editor.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <v-navigation-drawer app v-model="isOpen" clipped persistent absolute width="450" class="editor">
-    <fetch-manifest :setManifest="setManifest" :url="url" :setUrl="setUrl"></fetch-manifest>
+    <fetch-manifest :setManifest="setManifest" :url="url" :setUrl="setUrl" :isFetching="isFetching" :setIsFetching="setIsFetching"></fetch-manifest>
+    <div v-if="isFetching" class="fetching">
+      <p>Searching for manifest ... </p>
+      <v-progress-linear :indeterminate="true"></v-progress-linear>
+    </div>
+    <div v-if="!isFetching && !firstFetch && manifestEmpty" class="no-manifest">
```

Comment:
This seems a little bit excessive in a template, can you move it to a computed value?